ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -138,40 +139,19 @@ StackLayout {
             Item {
                 Layout.fillHeight: true
             }
+        }
 
-            RowLayout {
-                BannerPanel {
-                    objectName: "invitePeopleBanner"
-                    text: qsTr("Welcome to your community!")
-                    buttonText: qsTr("Invite new people")
-                    icon.name: "invite-users"
-                    onButtonClicked: root.inviteNewPeopleClicked()
-                }
-                Item {
-                   Layout.fillWidth: true
-                }
-                BannerPanel {
-                    objectName: "airdropBanner"
-                    visible: root.owned
-                    text: qsTr("Try an airdrop to reward your community for engagement!")
-                    buttonText: qsTr("Airdrop Tokens")
-                    icon.name: "airdrop"
-                    onButtonClicked: root.airdropTokensClicked()
-                }
-
-                Item {
-                   Layout.fillWidth: true
-                }
-
-                BannerPanel {
-                    objectName: "backUpBanner"
-                    visible: root.owned
-                    text: qsTr("Back up community key")
-                    buttonText: qsTr("Back up")
-                    icon.name: "objects"
-                    onButtonClicked: root.backUpClicked()
-                }
-            }
+        footer: OverviewSettingsFooter {
+            rightPadding: 64
+            leftPadding: 64
+            bottomPadding: 50
+            loginType: root.loginType
+            communityName: root.name
+            //TODO connect to backend
+            isControlNode: root.owned
+            onPrimaryButtonClicked: isControlNode = !isControlNode
+            //TODO update once the domain changes
+            onSecondaryButtonClicked: Global.openLink("https://help.status.im/en/status-communities/about-the-control-node-in-status-communities")
```

Comment:
Hmm, `en` language hardcoded?

Reply:
Good point! I've checked to see if other languages are available in documentation, but I'm getting 404 for anything different than `en`. Seems to me we're not supporting other languages.
@jorge-campo could you please confirm this? I'm using this URL: https://help.status.im/en/status-communities/about-the-control-node-in-status-communities

Reply:
Hey @caybro , @alexjba – For the MVP, we only support English. Additional languages will come much later.

Regarding the URL, please, be aware that our documentation website (Status Help, https://help.status.im/) is now being moved to the new Status web. I don't know the new URLs for any given doc on the new website. @prichodko, @marcelines, @felicio, guys, can you please help us here? 🙏

Reply:
1. Better extract this base help URL to `Constants`
2. The URL should always work, ie `https://help.status.im/status-communities/about-the-control-node-in-status-communities/` should redirect me to `https://help.status.im/en/status-communities/about-the-control-node-in-status-communities/` according to the user's preferred (based on the browser/app) language, or to English if it's the only option atm

Reply:
> > The URL should always work, ie https://help.status.im/status-communities/about-the-control-node-in-status-communities/ should redirect me to https://help.status.im/en/status-communities/about-the-control-node-in-status-communities/ according to the user's preferred (based on the browser/app) language, or to English if it's the only option atm
> 
> 404 It doesn't..

Yeah that needs to be fixed :) (on the server side)

Reply:
I think it's a good idea. Added an issue for this to the Web team to start a discussion. Might be a good moment for this discussion to happen as we're still in the process of moving the help pages to the new domain. Feel free to add your thoughts here https://github.com/status-im/status-web/issues/458

Reply:
> Regarding the URL, please, be aware that our documentation website (Status Help, https://help.status.im/) is now being moved to the new Status web.

Added a follow-up task for this. Once the new domain is up and running we'll change the URL - #11451

> Better extract this base help URL to Constants

Good point! Moved to Constants

Reply:
> > > The URL should always work, ie https://help.status.im/status-communities/about-the-control-node-in-status-communities/ should redirect me to https://help.status.im/en/status-communities/about-the-control-node-in-status-communities/ according to the user's preferred (based on the browser/app) language, or to English if it's the only option atm
> > 
> > 
> > 404 It doesn't..
> 
> Yeah that needs to be fixed :) (on the server side)

This will work as expected in the coming weeks once Status Help is integrated into the new Status website.